runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useProcessActionVariables.ts

```diff
@@ -87,9 +142,17 @@ const useProcessActionVariables = () => {
             .filter((item: ActionVariableObject[]) => item.length > 0)
             .flat();
 
-        return { inputActionVariables, outputActionVariables };
-    }, [allActionsWithVariables, canvas]);
+        if(!selectedElementParentId) return { inputActionVariables, outputActionVariables };
+            
+        const ancestorScopes = getAncestorScope(ProcessScopes, selectedElementParentId);
```

Comment:
u mean parent scope? ancestor does not suit here imo

Reply:
done